openedx/features/course_experience/views/course_updates.py

```diff
@@ -48,17 +51,29 @@ def render_to_fragment(self, request, course_id=None, **kwargs):
         course_url_name = default_course_url_name(request)
         course_url = reverse(course_url_name, kwargs={'course_id': unicode(course.id)})
 
-        # Fetch the updates as HTML
-        updates_html = get_course_info_section(request, request.user, course, 'updates')
+        # Fetch all of the updates
+        info_module = get_course_info_section_module(request, request.user, course, 'updates')
+        ordered_updates = self.order_updates(info_module.items)
```

Comment:
I'll see if I can find courses with each format to put on your sandbox so we can see how it works.